docs/src/main/asciidoc/azure-support-overview-concept.adoc

```diff
@@ -0,0 +1,117 @@
+////
+This document is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="azure-support-overview-concept"]
+= Quarkus on Microsoft Azure cloud platform
+include::_attributes.adoc[]
+:categories: cloud
+
+Quarkus provides support and capabilities to help you to build cloud-native applications that run on Microsoft Azure. Quarkus also provides support for building serverless Java functions by using Microsoft Azure Functions.
+You can deploy the container image of a Quarkus application that you have developed locally to the Microsoft Azure cloud platform.
+You can also entirely build, host, and run a Quarkus application on Azure by using some of the key developer services for Azure.
+
+== Microsoft Azure
+
+link:https://learn.microsoft.com/en-us/azure/?product=popular[Microsoft Azure] is a versatile and flexible cloud platform, which you can use to host your Quarkus applications.
+Azure supports various programming languages, complex architectures and frameworks, including Java and Quarkus.
+Azure also fully supports the Apache Maven Java development tool.
+You can use several link:https://learn.microsoft.com/en-us/azure/developer/intro/azure-developer-key-services[Azure key services for developers] to build and extend your Quarkus applications.
+You can also integrate and deploy Quarkus with the serverless environment of link:https://learn.microsoft.com/en-us/azure/azure-functions/[Azure Functions].
+
+Microsoft Azure also provides tools and services that integrate with Quarkus to make it easier to develop, deploy, and manage your Java applications on its cloud platform. For example:
+
+* link:[Azure DevOps] provides CI/CD pipelines for building and deploying Quarkus applications.
+* link:[Azure Monitor] provides capabilities for you to use to monitor and troubleshoot these applications.
+
+== Quarkus extensions for Microsoft Azure
+
+Quarkus provides the following extensions to make it easier to integrate your existing Quarkus applications with Microsoft Azure:
```

Comment:
These two are different from the others in that it is part of funky which is abstraction available in quarkus that lets you use serverless function style programming while mapping it to jaxrs existing programming model. Ie. Allowing to be portable. 

Azure functions is one of the supported modes. Amazon lambda is another.

Reply:
I created a new reference topic [Quarkus extensions for Azure integration](https://quarkus-pr-main-32823-preview.surge.sh/version/main/guides/azure-extensions-reference)  to provide guidance on the extensions and links to any existing resources we have that explain a bit about when and how to use them.